openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx

```diff
@@ -263,11 +266,36 @@ const TestSuitePipelineTab = () => {
     }
   };
 
+  const getAirflowStatus = () => {
+    return new Promise<void>((resolve, reject) => {
+      checkAirflowStatus()
+        .then((res) => {
+          if (res.status === 200) {
+            resolve();
+          } else {
+            reject();
+          }
+        })
+        .catch(() => reject());
+    });
+  };
```

Comment:
It's too repetitive please make single function to check status

Reply:
@chirag-madlani  changes updated.